specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualRouter.json

```diff
@@ -0,0 +1,699 @@
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Request successful. Resource with the specified name does not exist"
+          },
+          "200": {
+            "description": "Delete successful."
+          }
+        },
+        "x-ms-examples": {
+          "Delete VirtualRouter": {
+            "$ref": "./examples/VirtualRouterDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "get": {
+        "tags": [
+          "VirtualRouters"
+        ],
+        "operationId": "VirtualRouters_Get",
+        "description": "Gets the specified Virtual Router.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a Virtual Router resource.",
+            "schema": {
+              "$ref": "#/definitions/VirtualRouter"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get VirtualRouter": {
+            "$ref": "./examples/VirtualRouterGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "VirtualRouters"
+        ],
+        "operationId": "VirtualRouters_UpdateTags",
```

Comment:
>VirtualRouters_UpdateTags [](start = 24, length = 25)

Just  VirtualRouters_Update, as we could always expand this to patch other properties too